mentoring/group-mentee-guide.md

```diff
@@ -8,15 +8,15 @@ During the three month period, you will work on a path to the next level of memb
 
 Familiarize yourself with the [community membership requirements doc](/community-membership.md) including the requirements needed for your specific journey. You’ll receive sponsorship from the mentors and other people you've worked on issues/PRs with upon successful completion of the program. 
 
-## Expectations  
-You understand the requirements to become $member_role (ex: member->reviewer) 
-Good standing throughout the program and uphold our code of conduct
-Attend two bi-weekly standups a month
-The program is at your pace but these meetings will discuss your wins and blocks
-These meetings will also cover development area topics, suggested activities, and groomed issues
-Be helpful to your peers in the cohort
-Give back and be a mentor in a future cohort
-Mutual respect from mentors and peers who are in the cohort with you
+## Expectations    
```

Comment:
whitespace

Reply:
I'm not seeing the whitespace issue - is it between two words?

Reply:
After "## Expectations" I see 4 spaces. Felt unnecessary, but I might be wrong.